static/sass/print.scss

```diff
@@ -81,6 +88,7 @@ footer,
 .p-navigation,
 .global-nav__dropdown-overlay,
 .global-nav,
+.global-nav__header-list,
```

Comment:
Shouldn't this be within `.global-nav`?

Reply:
the text remained at an artefact in the footer of that page - what to explore?